src/middleware/index.js

```diff
@@ -2,6 +2,8 @@ const notFound = require('./not-found-handler');
 const addLogger = require('./logger');
 
 module.exports = (app) => {
+	// eslint-disable-next-line no-console
+	console.info('register middleware');
```

Comment:
Welchen Nutzen ziehen wir aus diesem Log?

Reply:
Wir wissen ab jetzt das die middleware nicht / oder an der falschen Stellen eingebaut ist und daher nicht ausgeführt wird. 
Ist für mich ähnlich dem level angesetzt wie "connect to database host", "version file existiert nicht". Man braucht die Info nicht, aber sie kann ein auch mal viel Zeit sparen und kostet uns keine.

Reply:
Steht da der Logger noch nicht zur Verfügung? die anderen Nachrichten, wie connect to database host, geht ja auch über den logger, dafür muss man hier keine console.info benutzen...